contracts/governance/scripts/DeployAddOperator.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.11;
+
+import "../../interfaces/INestedFactory.sol";
+import "../../interfaces/IOperatorResolver.sol";
+import "../../abstracts/MixinOperatorResolver.sol";
+
+contract DeployAddOperator {
+    struct tupleOperator {
+        bytes32 name;
+        bytes4 selector;
+    }
+
+    /// @notice Deploy and add operators
+    /// @dev One address and multiple selectors/names
+    /// @param nestedFactory The NestedFactory address
+    /// @param bytecode Operator implementation bytecode
+    /// @param operators Array of tuples => bytes32/bytes4 (name and selector)
+    function deployAddOperator(
+        address nestedFactory,
+        bytes memory bytecode,
+        tupleOperator[] memory operators
```

Comment:
It's a bit confusing:
- method name is singular
- one bytecode as param

but:

- `operators` param as an array - plural

Consider renaming `operators`

Reply:
In the `OperatorResolver`, an Operator is an `address` (implementation) and a `bytes4` (selector), linked to a `bytes32` (name).

The `bytecode` param will be deployed and as a result, the implementation (address). 
So, the `operators` is the array with `name` and the `selector`, it is really an operator list to add.

I don't know how we could rename `operators`. Any idea ?

Reply:
I would remove tuple from the `tupleOperator` name, it is explicit by looking at the type, no ?
Maybe name the struct `operatorFunction` and the array `operatorFunctions`  ?

Reply:
I understand how it works, then the method name is inaccurate, and should be plural.

Either the operator is the contract, or it's the address + selector + bytes. It just needs to be consistent across the protocol

Reply:
@adrien-supizet I misunderstood, I'm updating to `DeployAndAddOperators`